src/mir/passes.md

```diff
@@ -53,6 +53,18 @@ want to use a [MIR visitor]. MIR visitors are a handy way to walk all
 the parts of the MIR, either to search for something or to make small
 edits.
 
+### MIR optimization levels
+
+MIR optimizations can come in various levels of readiness. Experimental
+optimizations may cause miscompilations, or slow down compile times.
+However, these passes are complex enough to be included in nightly builds
+even in these states, to enable validating them or otherwise gather feedback.
+To enable working with slow or otherwise experimental optimization passes,
+you can specify the `-Z mir-opt-level` debug flag. You can find the
+definitions of the levels [here][1]. If you are developing a MIR pass and
```

Comment:
```suggestion
definitions of the levels in the [compiler MCP]. If you are developing a MIR pass and
```
and the link below updated to use `[compiler MCP]:` instead of [1]: